tests/test_pca.py

```diff
@@ -481,3 +519,27 @@ def test_pca_layer():
     )
     np.testing.assert_equal(X_adata.obsm["X_pca"], layer_adata.obsm["X_pca"])
     np.testing.assert_equal(X_adata.varm["PCs"], layer_adata.varm["PCs"])
+
+
+@needs.dask
+@pytest.mark.skipif(
+    pkg_version("anndata") < Version("0.10"),
+    reason="Old AnnData doesn’t have dask test helpers",
+)
```

Comment:
```suggestion
# Skipping this test during min-deps testing shouldn't be an issue because the sparse-in-dask feature is not available on anndata<0.10 anyway
@pytest.mark.skipif(
    pkg_version("anndata") < Version("0.10"),
    reason="Old AnnData doesn’t have dask test helpers",
)
```

Reply:
It took me a bit of time to come up with that explanation, so would be good for the next person